nestkernel/mpi_manager.h

```diff
@@ -294,6 +341,18 @@ class MPIManager : public ManagerInterface
   unsigned int send_recv_count_spike_data_per_rank_;
   unsigned int send_recv_count_target_data_per_rank_;
 
+  std::vector< int > recv_counts_secondary_events_in_int_per_rank_; //!< how many secondary elements (in ints) will be
+                                                                    //received from each rank
+  std::vector< int >
+    send_counts_secondary_events_in_int_per_rank_; //!< how many secondary elements (in ints) will be sent to each rank
+
+  std::vector< int > recv_displacements_secondary_events_in_int_per_rank_; //!< offset in the MPI receive buffer (in
+                                                                           //ints) at which elements received from each
+                                                                           //rank will be written
+  std::vector< int > send_displacements_secondary_events_in_int_per_rank_; //!< offset in the MPI send buffer (in ints)
+                                                                           //from which elements send to each rank will
+                                                                           //be read
```

Comment:
For Doxygen to see all documentation lines, they all need to start with `!<`.
```suggestion
  std::vector< int > recv_counts_secondary_events_in_int_per_rank_; //!< how many secondary elements (in ints) will be
                                                                    //!< received from each rank
  std::vector< int >
    send_counts_secondary_events_in_int_per_rank_; //!< how many secondary elements (in ints) will be sent to each rank

  std::vector< int > recv_displacements_secondary_events_in_int_per_rank_; //!< offset in the MPI receive buffer (in
                                                                           //!< ints) at which elements received from each
                                                                           //!< rank will be written
  std::vector< int > send_displacements_secondary_events_in_int_per_rank_; //!< offset in the MPI send buffer (in ints)
                                                                           //!< from which elements send to each rank will
                                                                           //!< be read
```